### DISCUSSION TITLE
Multi-Segment Mode

### DISCUSSION THREAD

**Discussion by bgwilkin** [awawa-dev/HyperSerialEsp8266#40] [Q&A] [answered]
Is multi-segment mode available for this version? In the README I see that it says it can be turned on, but in the platformio.ini file it says the pinout can't be changed, which implies you couldn't add a second pinout for the second segment.

**Comment by awawa-dev** [comment, marked as answer]
Sorry, that small part of compiling manual was copy-pasted from HyperSerialESP32. No, no support for multi-segment for esp8266 because of the hardware limitations: consider rp2040/pico board for even lower price (or buy more expensive pico with already built-in level shifter).
